__tests__/topics.test.js

```diff
@@ -0,0 +1,52 @@
+//const db = require("../db/data/test-data/topics");
+const app = require("../app");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const db = require("../db/connection")
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+	test("respond with STATUS 200.", () => {
+		return request(app)
+			.get("/api/topics")
+			.expect(200)
+		});
+	test("status 200: request should respond with an array of topic objects", () => {
+		return request(app)
+		.get("/api/topics")
+		.expect(200)
+		.then((topics) => {
+			expect(topics.body).toBeInstanceOf(Object);
+		});
+	});	
+	test("should return slug and description properties.", () => {
```

Comment:
This final test is proving everything asserted in the tests above, meaning this one test would have done the job by itself